services/book_browsing_and_sorting/book_api.py

```diff
@@ -0,0 +1,54 @@
+from flask import Flask, jsonify, request
+import mysql.connector
+
+app = Flask(__name__)
+
+# Database configuration
+db_config = {
+    'host': 'ls-b2f10f0f8d1f46949bc16b2a5608934e887eb6b0.c1zf3hrzxwhy.us-east-2.rds.amazonaws.com',
+    'user': 'team_8',
+    'password': 'Ankit-Bonnie-082821!',
```

Comment:
@elle-concetta I think we should put the password in a gitignore so it's not exposed.